### DISCUSSION TITLE
Geographic Sales Insights

### DISCUSSION THREAD

**Discussion by SrinivasGajawada** [amzn/selling-partner-api-models#616] [Ideas]
Hi Team -
Could you please help us to make the API development ready for Geographic Sales insight .
we need this for our critical reporting, so it would be helpful if expedite the process

thank you!

**Comment by krithic15** [comment]
Hi @SrinivasGajawada,

Thank you for your suggestion. Can you please let us know if this request is related to sales on Vendor account? And could you name the report/ API where you want to see Geographic sales insight?

Best,
Krithi, Solutions Architect
Selling Partner Developer Services

**Reply by r-add** [reply]
Hi @krithic15,

Is it possible to access Geographic Sales Insight through an API for a Vendor Central account?

**Reply by krithic15** [reply]
@r-add This report is not currently available via API but we have a request logged with our Product team as a lot of vendors have shown interest to access it via API. We do not have an ETA to share unfortunately and we appreciate your patience!

Best,
Krithi, Solutions Architect
Selling Partner Developer Services
